docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md

```diff
@@ -177,7 +177,29 @@ vbc Example1.vb
+  
+Each culture has only one parent, which is defined by the <xref:System.Globalization.CultureInfo.Parent%2A?displayProperty=nameWithType> property, but a parent might have its own parent. The search for parent cultures stops when a culture's <xref:System.Globalization.CultureInfo.Parent%2A> property returns <xref:System.Globalization.CultureInfo.InvariantCulture%2A?displayProperty=nameWithType>; for resource fallback, the invariant culture is not considered a parent culture or a culture that can have resources.  
+  
+4. If the culture that was originally specified and all parents have been searched and the resource is still not found, the resource for the default (fallback) culture is used. Typically, the resources for the default culture are included in the main application assembly. However, you can specify a value of <xref:System.Resources.UltimateResourceFallbackLocation.Satellite> for the <xref:System.Resources.NeutralResourcesLanguageAttribute.Location%2A> property of the <xref:System.Resources.NeutralResourcesLanguageAttribute> attribute to indicate that the ultimate fallback location for resources is a satellite assembly, rather than the main assembly.  
+  
+    > [!NOTE]
+    >  The default resource is the only resource that can be compiled with the main assembly. Unless you specify a satellite assembly by using the <xref:System.Resources.NeutralResourcesLanguageAttribute> attribute, it is the ultimate fallback (final parent). Therefore, we recommend that you always include a default set of resources in your main assembly. This helps prevent exceptions from being thrown. By including a default resource file you provide a fallback for all resources, and ensure that at least one resource is always present for the user, even if it is not culturally specific.  
+  
+5. Finally, if the runtime doesn't find a resource for a default (fallback) culture, a <xref:System.Resources.MissingManifestResourceException> or <xref:System.Resources.MissingSatelliteAssemblyException> exception is thrown to indicate that the resource could not be found.  
+  
+ For example, suppose the application requests a resource localized for Spanish (Mexico) (the es-MX culture). The runtime first searches the global assembly cache for the assembly that matches es-MX, but doesn't find it. The runtime then searches the directory of the currently executing assembly for an es-MX directory. Failing that, the runtime searches the global assembly cache again for a parent assembly that reflects the appropriate fallback culture — in this case, es (Spanish). If the parent assembly is not found, the runtime searches all potential levels of parent assemblies for the es-MX culture until it finds a corresponding resource. If a resource isn't found, the runtime uses the resource for the default culture.    
```

Comment:
Does this (especially the global assembly cache) apply to .NET Core?

Reply:
It does not. GAC (Global assembly cache) does not exist in .NET Core.

Reply:
Ah, this is just a copy-and-paste error, @sdmaclea -- the paragraph is repeated earlier on line 95. So this paragraph should be removed. Given that .NET Core's probing is simpler than .NET Framework's, I don't think that we need a for example paragraph -- the steps in the probe are sufficient.